roles/ensure-gh/tasks/main.yaml

```diff
@@ -1,4 +1,4 @@
-- name: Check installed vault version
+- name: Check installed gh version
   command: vault version
   register: vault_installed_version
```

Comment:
installed `gh` version with `vault` cmd. Is it right?

Reply:
ugh, right, it's bit more